test/images/stub-device-plugin/pod.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
If we are not using these yml anywhere in the testcode, may I know the intention of adding these ymls? If answer is for reference then we should add a readme in the same folder with more information on how to use them

Reply:
@mkumatag Thanks for your comments. It is just for reference, and i will remove them as they are not used in the test code.